providers/opcua/resources/opcua.lr

```diff
@@ -1,4 +1,5 @@
-option go_package = "go.mondoo.com/cnquery/resources/packs/opcua"
```

Comment:
As discussed, this now also needs a license header.

Reply:
+1, resolved